scss/components/_tags-container.scss

```diff
@@ -1,7 +1,11 @@
+// holds about 40 characters of variable width text:
+$tag-max-width: 36ch;
```

Comment:
Why `ch`? We dont use it generally. Now there is no harm in using it, but any specific reason to use this instead of px?

Reply:
Also the comment should be on the right side.
`$tag-max-width: 36ch; //...`

Reply:
@deb1990 using `ch` because the width we want to achieve is based on number of characters, not an arbitrary pixel width.

As mentioned in the description, this is not a fixed-size font so we can't just put `40ch` and expect it to look good, but `36ch` is a good approximation and will output around 40 characters wide tags.